src/main/java/edu/greenblitz/robotName/commands/arm/Wrist/WristCommand.java

```diff
@@ -0,0 +1,15 @@
+package edu.greenblitz.robotName.commands.arm.Wrist;
+
+import edu.greenblitz.robotName.subsystems.Arm.Wrist;
+import edu.greenblitz.robotName.utils.GBCommand;
+
+public class WristCommand extends GBCommand {
+
+    protected Wrist wrist;
```

Comment:
Why protected and not private?

Reply:
because in yerosha you cant use it if it is private

Reply:
you can